DiceKit/ProbabilityMass.swift

```diff
@@ -8,28 +8,36 @@
 
 import Foundation
 
-public struct ProbabilityMass: ApproximatelyEquatable {
+// TODO: Find a proper place for this
+public struct ProbabilityMassConfig {
```

Comment:
I don't see why it needs to move from here. Its at the top of the file its relevant too, and its not large enough to warrant moving it to its own file imo.